camdecmpswks/procedures/update_mp_eval_status_and_reporting_freq.sql

```diff
@@ -1,8 +1,11 @@
--- PROCEDURE: camdecmpswks.update_mp_eval_status_and_reporting_freq()
+-- PROCEDURE: camdecmpswks.update_mp_eval_status_and_reporting_freq(numeric, character varying)
 
-DROP PROCEDURE IF EXISTS camdecmpswks.update_mp_eval_status_and_reporting_freq();
+DROP PROCEDURE IF EXISTS camdecmpswks.update_mp_eval_status_and_reporting_freq(numeric, character varying);
 
-CREATE OR REPLACE PROCEDURE camdecmpswks.update_mp_eval_status_and_reporting_freq()
```

Comment:
Are we sure that this version of the SP is not used in the [Change Data Capture (CDC) Process](https://github.com/US-EPA-CAMD/easey-docs/blob/develop/CAMDRunbook.md#32-change-data-capture-cdc)?

Reply:
Hmm I'm honestly not sure. I searched organization wide for the procedure and didn't find it anywhere else, but it could be somewhere I'm not privy to.

Reply:
I did the same search.  I am going to approve and then merge both tickets.